libcxx/test/libcxx/debug/extern-templates.sh.cpp

```diff
@@ -1,52 +0,0 @@
-//===----------------------------------------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-// This test checks that we retain extern template instantiation declarations
-// for members of <locale> even when the debug mode is enabled, which is
-// necessary for correctness. See https://llvm.org/D94718 for details.
-
-// UNSUPPORTED: !libcpp-has-legacy-debug-mode
-// UNSUPPORTED: no-localization
-// UNSUPPORTED: cant-build-shared-library
```

Comment:
We could also remove the `cant-build-shared-library` Lit feature.

Reply:
Thanks!